src/libs/ReportActionsUtils.ts

```diff
@@ -107,19 +110,23 @@ function isDeletedAction(reportAction: OnyxEntry<ReportAction | OptimisticIOURep
     return isLegacyDeletedComment || !!message[0]?.deleted;
 }
 
+function getReportActionMessage(reportAction: PartialReportAction) {
```

Comment:
Please specify the return type explicitly.

Here and in other functions you introduced.